sources/player/Player.gd

```diff
@@ -0,0 +1,75 @@
+extends KinematicBody2D
+
+
+class_name Player
+
+signal death
+
+export(float) var Gravity = 500
+export(float) var  RunMaxSpeed = 500
+export(float) var  JumpSpeed = 200
+export(float) var JumpMaxDuration = 0.5
+export(float) var  RunForce = 1000
+export(float) var StopForce = 1000
+
+var velocity: Vector2 = Vector2()
+var jumping: bool = false
+var on_air_time: float = 0
+
+func _ready():
+	pass
+
+
+#func _process(delta):
+#	pass
+
+func _physics_process(delta):
+	var force = Vector2(0, Gravity)
+
+	var walk = Input.get_action_strength("ui_right") - Input.get_action_strength("ui_left")
+	var jump = Input.is_action_pressed("ui_up")
+	
+	# while jumping, player can't change direction. Can be update later
+	if not jumping:
+		force.x = update_horizontal_force(walk, delta)
+		
+	velocity += force * delta
```

Comment:
Regarding move_and_slide: https://docs.godotengine.org/en/3.1/classes/class_kinematicbody2d.html#class-kinematicbody2d-method-move-and-slide You should not multiply it by delta, its already taken care of.